lldb/test/API/lang/swift/expression/error_missing_type/TestSwiftExpressionErrorMissingType.py

```diff
@@ -0,0 +1,15 @@
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+
+class TestSwiftExpressionErrorReportingy(TestBase):
+    NO_DEBUG_INFO_TESTCASE = True
+
+    @swiftTest
+    def test(self):
+        """Test an extra hint inserted by LLDB for missing module imports"""
+        self.build()
+        lldbutil.run_to_source_breakpoint(self, 'break here', lldb.SBFileSpec('main.swift'))
+
+        self.expect('expression s.e', error=True,  substrs=['module', 'import'])
```

Comment:
Maybe add more words here? Like the beginning of the note? "If the member name is spelled correctly"